src/vs/workbench/contrib/accessibility/browser/accessibilityContribution.ts

```diff
@@ -47,7 +51,7 @@ const configuration: IConfigurationNode = {
 			...baseProperty
 		},
 		[AccessibilityVerbositySettingId.InlineChat]: {
-			description: localize('verbosity.interactiveEditor.description', 'Provide information about how to access the inline editor chat accessibility help menu and alert with hints which describe how to use the feature when the input is focused'),
+			description: localize('verbosity.interactiveEditor.description', 'Provide information /Users/meganrogge/Repos/vscode/vscode/src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.tsabout how to access the inline editor chat accessibility help menu and alert with hints which describe how to use the feature when the input is focused'),
```

Comment:
?

Reply:
I copied a file path from the terminal and tried to paste it into the quickpick - seemingly it pasted in the editor instead and I didn't notice. thanks